docker_mcp/tools/containers.py

```
@@ -704,6 +692,7 @@
                 "conflicts": [conflict.model_dump() for conflict in conflicts],
                 "summary": summary,
                 "timestamp": datetime.now().isoformat(),
+                "cached": cache_hit,
```

Ruff (0.12.2)

Undefined name `cache_hit`

(F821)